PM updates and new TS files (#72)
mulesoft/docs-partner-manager: modules/ROOT/pages/ts-no-T1-acks.adoc

= Missing TA1 acknowledgments for my transmissions

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Anypoint Partner Manager sends TA1 acknowledgments back for any received transmission, if configured to do so. Your trading partner may contact you to tell you that they are not receiving TA1 acknowledgements.

Reviewer: ```suggestion
Anypoint Partner Manager sends TA1 acknowledgments for received transmissions if configured to do so. Your trading partner might contact you to tell you that they are not receiving TA1 acknowledgments.
```

Author: f

== Causes

Any of the following can lead to this type of error:

> . Your trading partners X12 settings are incorrectly configured do not send TA1’s back to the partner.

Reviewer: Grammatically incorrect. Please reread for accuracy:
```suggestion
. Your trading partners' X12 settings are incorrectly configured to not return TA1s.
```

Author: F

> . The endpoint to which you are sending that TA1’s back may be incorrectly configured.

Reviewer: ```suggestion
. The endpoint to which you are sending TA1s might be incorrectly configured.
```

== Solution

> In case you are not receiving TA1 configurations, please review the following configuration.

Reviewer: ```suggestion
If you are not receiving TA1 configurations, review the following configuration:
```

Author: f

> . From within your Anypoint Partner Manager environment (either Sandbox or Production) select *Partners* and Receive from <Partner_Name>.

Reviewer: ```suggestion
. From within your Anypoint Partner Manager environment (either Sandbox or Production), select *Partners* and *Receive from <Partner_Name>*.
```

Author: f

. Scroll to Validation and Acknowledgment settings and click *X12*.

> . Within the Acknowledgements section ensure that the option for ‘Send TA1 for each transmission from <partner name>’ is selected.

Reviewer: ```suggestion
. Within the Acknowledgements section, ensure that the option for ‘Send TA1 for each transmission from <partner name>’ is selected.
```

Author: f

> . Configure the Acknowledgement endpoint as in xref:endpoints.adoc[Endpoints]

Reviewer: ```suggestion
. Configure the Acknowledgement endpoint as in xref:endpoints.adoc[Endpoints].
```

Author: f

. Click *Save* when done.

> This enables TA1 acknowledgments for all transmissions received by the specific partner.

Reviewer: ```suggestion
This enables TA1 acknowledgments for all transmissions received by the named partner.
```

Author: will not fix
